pkg/models/system_intake.go

```diff
@@ -130,6 +130,8 @@ type SystemIntake struct {
 	DecisionNextSteps           null.String             `json:"decisionNextSteps" db:"decision_next_steps"`
 	RejectionReason             null.String             `json:"rejectionReason" db:"rejection_reason"`
 	AdminLead                   null.String             `json:"adminLead" db:"admin_lead"`
+	LastAdminNoteContent        null.String             `json:"lastAdminNoteContent" db:"last_admin_note_content"`
+	LastAdminNoteCreatedAt      *time.Time              `json:"lastAdminNoteCreatedAt" db:"last_admin_note_created_at"`
```

Comment:
I originally tried to use a new struct for this work like this:

```go
type SystemIntakeWithLastAdminNote struct {
    SystemIntake
    LastAdminNoteContent        null.String             `json:"lastAdminNoteContent" db:"last_admin_note_content"`
 	LastAdminNoteCreatedAt      *time.Time              `json:"lastAdminNoteCreatedAt" db:"last_admin_note_created_at"`
}
```

This worked well enough in the `storage` package, but when it came time to update the service and handler it ended up being a lot of changes because of the type change. I reverted that and just added the new fields to the existing struct considering that if we were to move to GraphQL, we would likely load the note in a different resolver anyway.